doc/nrf/libraries/caf/ble_bond.rst

```diff
@@ -0,0 +1,62 @@
+.. _caf_ble_bond:
+
+CAF: Bluetooth LE bond module
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |ble_bond| of the :ref:`lib_caf` (CAF) allows to introduce Bluetooth LE bonding to the application.
+The module also allows to delete bonds for the default Bluetooth local identity.
+
+Configuration
+*************
+
+Before using the module make sure that the following Kconfig options are enabled:
+
+* :kconfig:`CONFIG_BT_BONDABLE`
+* :kconfig:`CONFIG_BT_SETTINGS`
+* :kconfig:`CONFIG_CAF_SETTINGS_LOADER`
+* :kconfig:`CONFIG_CAF_BLE_COMMON_EVENTS`
+
+The device must be Bluetooth bondable and bonds must be stored in non-volatile memory (settings).
+The :ref:`caf_settings_loader` is used to load content of the settings area during boot.
+
+Enable the |ble_bond| module using :kconfig:`CONFIG_CAF_BLE_BOND`.
+This Kconfig option selects :kconfig:`CONFIG_CAF_BLE_BOND_SUPPORTED` option to indicate that Bluetooth LE bond functionality is supported by the application.
```

Comment:
```suggestion
This Kconfig option selects :kconfig:`CONFIG_CAF_BLE_BOND_SUPPORTED` option to indicate that the |ble_bond| functionality is supported by the application.
```

Reply:
Added missing `the`. We cannot use `|ble_bond|` here, because it expands to `Bluetooth LE bond module`

Reply:
Rephrased this one a bit.